.zuul.yaml

```diff
@@ -1,46 +1,39 @@
+# Copyright (c) 2024 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
 - job:
-    name: libvirtd_exporter:image:build
-    parent: vexxhost-build-docker-image
-    provides: libvirtd_exporter:image
+    name: libvirtd-exporter-build-container-image
+    parent: ci-build-container-image
     vars: &libvirtd_exporter_images
-      docker_images:
+      promote_container_image_job: libvirt-exporter-upload-container-image
+      container_images:
         - context: .
-          repository: vexxhost/libvirtd-exporter
+          repository: registry.atmosphere.dev
           arch:
             - linux/amd64
             - linux/arm64
+          tags:
+            - "{{ zuul.commit_id }}"
+            - "{{ zuul.tag is defined | ternary(zuul.tag, 'latest') }}"
 
 - job:
-    name: libvirtd_exporter:image:upload
-    parent: vexxhost-upload-docker-image
-    provides: libvird_exporter:image
+    name: libvirtd-exporter-upload-container-image
+    parent: ci-upload-docker-image
     vars: *libvirtd_exporter_images
 
 - job:
-    name: libvirtd_exporter:image:promote
-    parent: vexxhost-promote-docker-image
+    name: libvirtd-exporter-promote-container-image
+    parent: ci-promote-container-image
     vars: *libvirtd_exporter_images
 
-- job:
-    name: libvirtd_exporter:github-mirror
-    parent: vexxhost-upload-git-mirror
-    vars:
-      git_mirror_repository: vexxhost/libvirtd_exporter
-
 - project:
-    templates:
-      - publish-opendev-tox-docs
+    merge-mode: squash-merge
     check:
       jobs:
-        - golangci-lint
```

Comment:
Please keep running the golangci-lint job for now.

Reply:
done